pkg/agent/util/net_windows.go

```diff
@@ -425,26 +431,6 @@ func EnableRSCOnVSwitch(vSwitch string) error {
 	return nil
 }
 
-func enableHNSOnOVS(hnsNet *hcsshim.HNSNetwork) error {
-	// Release OS management for HNS Network if Hyper-V is enabled.
-	hypervEnabled, err := WindowsHyperVEnabled()
-	if err != nil {
-		return err
-	}
-	if hypervEnabled {
-		if err := RemoveManagementInterface(LocalHNSNetwork); err != nil {
```

Comment:
Is this the step that removes the vnic we use since this PR?
Do we still need the two functions? WindowsHyperVEnabled and RemoveManagementInterface

Reply:
RemoveManagementInterface is not needed any more (and I would remove it), but WindowsHyperVEnabled is used in the integration test for HyperV features check on the setup.